gfsphysics/GFS_layer/GFS_typedefs.F90

```diff
@@ -6971,6 +6987,11 @@ subroutine interstitial_print(Interstitial, Model, mpirank, omprank, blkno)
     write (0,*) 'sum(Interstitial%dudt_mtb        ) = ', sum(Interstitial%dudt_mtb        )
     write (0,*) 'sum(Interstitial%dudt_ogw        ) = ', sum(Interstitial%dudt_ogw        )
     write (0,*) 'sum(Interstitial%dudt_tms        ) = ', sum(Interstitial%dudt_tms        )
+!-- GSD drag suite
```

Comment:
If these are only used for the GSD drag suite, then please print them only if the drag suite is used.